packages/rocketchat-bot-helpers/server/index.js

```diff
@@ -0,0 +1,77 @@
+
+// config to extend query / return attributes
+const defaults = {
+	userFields: {
+		_id: 1,
+		name: 1,
+		username: 1,
+		status: 1,
+		emails: 1,
+		language: 1
+	},
+	onlineQuery: { 'status': { $ne: 'offline' } },
+	userQuery: { 'roles': { $not: { $all: ['bot'] } } }
+};
+
+/**
+ * BotHelper helps bots
+ * keeping Rocket.chat stuff in meteor package allows clean logic separation in bot scripts
+ * "private" properties use meteor collection cursors, so they stay reactive
+ * "public" "properties" use getters to fetch and filter collections as array
+ */
+class BotHelper {
+	constructor(options = {}) {
+		this.config = _.extend({}, defaults, options);
+		this._allUsers = Meteor.users.find(this.config.userQuery, { fields: this.config.userFields });
```

Comment:
There is a security issue here, anyone can call this method and get all user's data, I can agree with some fields like name, username, utcOffset, etc.